sdk/keyvault/keyvault-secrets/review/keyvault-secrets.review.api.md

```diff
@@ -0,0 +1,151 @@
+## API Report File for "@azure/keyvault-secrets"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import { HttpClient } from '@azure/core-http';
+import { HttpPipelineLogger } from '@azure/core-http';
+import * as msRest from '@azure/core-http';
+import { PagedAsyncIterableIterator } from '@azure/core-paging';
+import { PageSettings } from '@azure/core-paging';
+import { RequestOptionsBase } from '@azure/core-http';
+import { ServiceClientOptions } from '@azure/core-http';
+import { TokenCredential } from '@azure/core-http';
+
+// @public
+export interface DeletedSecret extends Secret {
+    readonly deletedDate?: Date;
+    recoveryId?: string;
+    readonly scheduledPurgeDate?: Date;
+}
+
+// @public
+export type DeletionRecoveryLevel = "Purgeable" | "Recoverable+Purgeable" | "Recoverable" | "Recoverable+ProtectedSubscription";
+
+// @public
+export interface GetSecretOptions {
+    requestOptions?: msRest.RequestOptionsBase;
+    version?: string;
+}
+
+// @public
+export interface GetSecretsOptions {
```

Comment:
Should this be called `ListSecretsOptions`? For two reasons: both that `GetSecretOptions` is too similar to `GetSecretOptions` and that it doesn't match the `list` verb that all the consuming methods have. If those methods are renamed as discussed below, I think the type here should align with the methods that use it (`ListSecretPropertiesOptions`), just as the method that uses `GetSecretOptions` is called `getSecret`.

Or, if we like my suggestion to use a wrapper type with `requestOptions? : msRest.RequestOptionsBase` as in keyvault-keys, then we could just use that type instead of this one.